lib/lamby/rack_http.rb

```diff
@@ -38,7 +38,7 @@ def env_base
 
     def env_headers
       super.tap do |hdrs|
-        if event['cookies'].present?
+        if !event['cookies'].to_s.empty?
```

Comment:
I thought `event['cookies']` could be an array? See here https://github.com/customink/lamby/blob/master/test/test_helper/events/http_v2.rb

Reply:
You're right - good catch!

https://github.com/awsdocs/aws-lambda-developer-guide/blob/3c88388346dd2c6b31723f32e432b5414d7b7d26/sample-apps/nodejs-apig/event-v2.json#L6-L9

Reply:
Reminder, it will be nil on HTTP v1. Not sure if the array is there all the time too or just empty on HTTP v2. So maybe some existential operator usage here too.

Reply:
Okay, I've pushed a commit that fixes this. Depending on the event from ApiGateway, we'll either take the `event['cookies']` value, or default it to the empty array.

i.e.

```
def cookies
  event['cookies'] || []
end
```